.gitignore

```
@@ -6,7 +6,7 @@ node_modules/
 deployments/
 env/
 .env
-dist/
+/dist
```

Can you undo this change?
```suggestion
dist/
```

---

Can you undo this change as well?